src/buildCommon.js

```diff
@@ -165,9 +165,11 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     // Math mode or Old font (i.e. \rm)
     const isFont = mode === "math" || (mode === "text" && options.font);
     const fontOrFamily = isFont ? options.font : options.fontFamily;
-    if (text.charCodeAt(0) === 0xD835) {
+    const [wideFontName, wideFontClass] = text.charCodeAt(0) !== 0xD835
+      ? ["", ""]
+      : wideCharacterFont(text, mode);
+    if (wideFontName.length > 0) {
```

Comment:
It would be nice to avoid the array creation in the negative case. How about this:
```js
let wideFontName, wideFontClass;
if (text.charCodeAt(0) === 0xD835) {
  [wideFontName, wideFontClass] = wideCharacterFont(text, mode);
}
if (wideFontName) {
```

Reply:
I like it.